src/components/TodoAdd/TodoAdd.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todos: Todo[]
+  onSetTodos: (newValue: Todo[]) => void
+}
+
+export const TodoAdd = React.memo<Props>(({ todos, onSetTodos }) => {
+  const [title, setTitle] = useState('');
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (title.trim()) {
+      onSetTodos([...todos, {
+        id: +new Date(),
```

Comment:
suggestion:
new Date is ok here but if you work with API it will create too long id

here is a nice package to generate id's: https://www.npmjs.com/package/uuid

Reply:
Yeah, but it was  suggested in the task description